lib/irc.js

```diff
@@ -633,6 +634,13 @@ function Client(server, clientNick, opt) {
                 // currently only handle ACK sasl responses
```

Comment:
Nitpick: this no longer only handles ACKs for sasl, so could you amend this? ("currently only handles ACK sasl and extended-join", for example)